### PR TITLE
Fix bug donation

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by BastienSaulnier — APPROVED** [review]

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: frontend/src/pages/CampaignInfos.jsx [hunk 1]
```diff
@@ -119,8 +132,16 @@ export default function CampaignInfos(props) {
...
+            participation à la campagne <span>{campaignInfo.name} !</span>
           </p>
+          {donor.donation_value < 1 ? (
+            ""
+          ) : (
+            <p>
+              {" "}
+              À hauteur de <span>{donor.donation_value} €</span> !
+            </p>
+          )}
```
Comment (LoicBrassart) [inline frontend/src/pages/CampaignInfos.jsx]:
```
{
    donor.donation_value>0 && 
    <p>A hauteur de <span>{donor.donation_value} €</span> !
}
```

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Left a better writing for a bit of code

File: frontend/src/pages/CampaignInfos.jsx (hunk 1 above)
Comment (LeoDiduch) [reply frontend/src/pages/CampaignInfos.jsx]: Ok change this right now

**Review by BastienSaulnier — APPROVED** [review]
SnE changes were made

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**